rholang/examples/linking/packages/MakeBrandPair.rho

```diff
@@ -0,0 +1,18 @@
+//based on sealer/unsealer pattern in http://erights.org/elib/capability/ode/ode-capabilities.html
+export MakeBrandPair in {
+  contract MakeBrandPair(return) = {
+    new seal, unseal, key in {
+      contract seal(data, return) = {
+        new sealedData in {
+          contract sealedData(input, return) = {
+            match input == key with true => { return(data) }
+            _ => { return(Nil) }
```

Comment:
Since on the blockchain this pattern is only useful for storing unforgeable names, `Nil` is a reasonable return value.  If it's intended to be used in system contracts to store arbitrary data, you may want to say `return([data])` so you can distinguish a stored `Nil` from a `not present` result.